Objects/setobject.c

```diff
@@ -1725,11 +1753,18 @@ set_issubset(PySetObject *so, PyObject *other)
         Py_RETURN_FALSE;
 
     while (set_next(so, &pos, &entry)) {
+        PyObject *key = entry->key;
+        Py_INCREF(key);
         rv = set_contains_entry((PySetObject *)other, entry->key, entry->hash);
-        if (rv < 0)
+        if (rv < 0) {
+            Py_DECREF(key);
```

Comment:
For this loop, I'd move the decref to just after `set_contains_entry`; it'll make the ref handling easier

Reply:
Fixed, thanks.